aiida/cmdline/commands/work.py

```diff
@@ -8,12 +8,19 @@
 # For further information please visit http://www.aiida.net               #
 ###########################################################################
 import click
+import logging
 from tabulate import tabulate
 from aiida.cmdline.commands import work, verdi
 from aiida.cmdline.baseclass import VerdiCommandWithSubcommands
 
 CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
 
+LOG_LEVELS = [
+    levelname for levelname in [
+        logging.getLevelName(i) for i in range(51)
```

Comment:
Ah, that should be ``logging.CRITICAL + 1``. I don't think we need to support if the user decides to add even higher levels, since that's theoretically also possible.